extensions/pritunl/src/types/preferences.ts

```diff
@@ -0,0 +1,8 @@
+import { getPreferenceValues } from "@raycast/api";
+import { Application } from "@raycast/api";
+
+export const preferences: {
+  connectionId: string;
+  connectionPin: string;
+  pritunlApp: Application;
+} = getPreferenceValues();
```

Comment:
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`x